intro/custom-domains.md

```diff
@@ -1,20 +1,21 @@
 # Custom Domains
 
-###### This is an overview of how WeDeploy manage custom domains for your projects.
+###### This is an overview of how WeDeploy manages custom domains for your projects.
 
 <!-- <article id="introduction"> -->
 
 ## Introduction
 
-In order to make your app accessible right away, WeDeploy makes any project accessible via its project domain, wich looks like `<serviceID>.<projectID>.wedeploy.io`. Let's say you have a project named `project` with a service named `web` the WeDeploy project domain would be `web.project.wedeploy.io`. Every project in WeDeploy supports one or many custom domains.
+To make your app accessible right away, WeDeploy makes any project accessible via its project domain; wich looks like `<serviceID>.<projectID>.wedeploy.io`. Let's say you have a project named `project` with a service named `web` the WeDeploy project domain would be `web.project.wedeploy.io`. Every project in WeDeploy supports one or many custom domains.
```

Comment:
@eduardolundgren it was a suggestion from `https://grammarly.com/` I can change back if you want.